podcasts/TranscriptsViewController.swift

```diff
@@ -390,12 +502,48 @@ extension TranscriptsViewController: UIScrollViewDelegate {
 extension TranscriptsViewController: TranscriptSearchAccessoryViewDelegate {
     func doneTapped() {
         dismissSearch()
+        resetSearch()
         searchView.removeFromSuperview()
     }
 
     func searchButtonTapped() {
         becomeFirstResponder()
     }
+
+    func search(_ term: String) {
+        guard !term.isEmpty else {
```

Comment:
As much as I love `guard` I would avoid to use it as double negative. In these cases `if` is more readable, as `if term.isEmpty`

Reply:
Done in ee793f555a4e572d99d31f421cb5585a1b34e94d